src/connectors/sort-by/connectSortBy.js

```diff
@@ -103,26 +103,26 @@ export default function connectSortBy(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.sortBy',
 
-      init({ helper, instantSearchInstance }) {
-        const initialIndex = helper.state.index;
-        const isInitialIndexInItems = find(
+      init({ helper, instantSearchInstance, parent }) {
+        const currentIndex = helper.state.index;
+        const isCurrentIndexInItems = find(
```

Comment:
is this dead code eliminated in the prod build? it seems to be just for dev mode

Reply:
That's worth checking, but really related to this PR.

Reply:
I have checked, and both the warning and the find are present in the prod build